development/templates/development/match_details.html

```diff
@@ -4,6 +4,14 @@
 <div class="container-fluid">
     {% if data %}
     <h1>Match details 
+
+        <select class="form-control" name="valid_move" id="valid_move">
+            <optgroup label="Validez">
```

Comment:
Label doesn't match Screenshot. Also Label should be in english

Reply:
My mistake, After I took the screenshot, I made changes. I'm fixing it